sdks/java/io/jdbc/src/test/java/org/apache/beam/sdk/io/jdbc/JdbcTestDataSet.java

```diff
@@ -38,26 +37,8 @@
   private static final Logger LOG = LoggerFactory.getLogger(JdbcTestDataSet.class);
   public static final String[] SCIENTISTS = {"Einstein", "Darwin", "Copernicus", "Pasteur", "Curie",
       "Faraday", "McClintock", "Herschel", "Hopper", "Lovelace"};
-  /**
-   * Use this to create the read tables before IT read tests.
-   *
-   * <p>To invoke this class, you can use this command line:
-   * (run from the jdbc root directory)
-   * mvn test-compile exec:java -Dexec.mainClass=org.apache.beam.sdk.io.jdbc.JdbcTestDataSet \
-   *   -Dexec.args="--postgresServerName=127.0.0.1 --postgresUsername=postgres \
-   *   --postgresDatabaseName=myfancydb \
-   *   --postgresPassword=yourpassword --postgresSsl=false" \
-   *   -Dexec.classpathScope=test
-   * @param args Please pass options from IOTestPipelineOptions used for connection to postgres as
-   * shown above.
-   */
-  public static void main(String[] args) throws SQLException {
-    PipelineOptionsFactory.register(IOTestPipelineOptions.class);
-    IOTestPipelineOptions options =
-        PipelineOptionsFactory.fromArgs(args).as(IOTestPipelineOptions.class);
-
-    createReadDataTable(getDataSource(options));
-  }
+  public static final long EXPECTED_ROW_COUNT = 1000L;
+  public static final String EXPECTED_HASH_CODE = "7d94d63a41164be058a9680002914358";
```

Comment:
I ended up abstracting out the TestRow class - it is actually not Jdbc specific, rather it is constructed so that it could conceivably be used to test any IO that can store a number and a string together. 

However, I played around with abstracting out the verifications and it didn't end up feeling right - it violated the "you should be able to read the test and immediately tell what's going on." 

What I did end up with was removing a lot of the cruft *around* the test and moving that to a shared location (TestRow.)

Let me know what you think.